src/main/java/domain/Number.java

```diff
@@ -0,0 +1,21 @@
+package domain;
+
+public class Number {
+    private int value;
```

Comment:
```suggestion
    private final int value;
```
불변으로 처리할 수 있을 것 같네요 😄